07LessonSeven/homework/index.html

```diff
@@ -1,10 +1,38 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+    <title>Profession Portal</title>
+    <link rel="stylesheet" href="css/style.css">
 </head>
-<body>
 
+<body>
+    <main>
+        <header>
+            <h1>Profession Portal</h1>
+        </header>
+        <nav>
+            <a href="#">Home</a>
+            <a href="#">Companies</a>
+            <a href="#">Positions</a>
+        </nav>
+        <article class="hire">
+            <h3> Hire an employee</h3>
```

Comment:
Try to avoid unnecessary spaces in your text elements - should be `<h3>Hire an employee</h3>`